bentoml/_internal/bento/gen.py

```diff
@@ -173,6 +173,7 @@ def generate_dockerfile(
         extensions=["jinja2.ext.do", "jinja2.ext.loopcontrols", "jinja2.ext.debug"],
         trim_blocks=True,
         lstrip_blocks=True,
+        autoescape=True,  # https://docs.huihoo.com/google-ctemplate/auto_escape.html
```

Comment:
Hm, seems like this is made for escaping HTML / JS and not dockerfiles?

Reply:
i guess but bandit is complaining about it.

Reply:
oh i ran [bandit](https://github.com/PyCQA/bandit) locally. I will remove this for now.